app/src/main/java/org/theotech/ceaselessandroid/CeaselessApplication.java

```diff
@@ -41,21 +38,16 @@ private void createNotificationChannel() {
     public void onCreate() {
         super.onCreate();
         // crashlytics
-        if (!BuildConfig.DEBUG) {
-            Fabric.with(this, new Crashlytics());
-        }
+//        Crashlytics crashlyticsKit = new Crashlytics.Builder()
```

Comment:
we should clean this up if we don't need it.

Reply:
Wanted to keep this around till Crashlytics is actually tested.